pkg/operator/bootstrap.go

```diff
@@ -119,7 +119,7 @@ func (c *Controller) Bootstrap() error {
 		cr.Spec.HTTPSecret = fmt.Sprintf("%x", string(secretBytes[:]))
 	}
 
-	driver, err := storage.NewDriver(cr.Name, c.params.Deployment.Namespace, &cr.Spec.Storage)
+	driver, err := storage.NewDriver(&cr.Spec.Storage)
```

Comment:
i can still see cases where a driver might need this information.  e.g. the filesystem driver which has to create PVCs in the registry namespace.

Reply:
Can't we just use a constant in the api for that?  I believe we are doing it elsewhere in the code base.

Reply:
Do you want me to close this pr then?

Reply:
yes let's hold off on further driver interface refactoring until we have more implementations that really tell us what shape we need it to take on.